### ISSUE TITLE
What is the scope of the data?

### ISSUE THREAD

**Issue by cmeeren** [andris9/jStorage#98]
Forgive me if I have overlooked this somewhere, but what is the scope of the stored data? Can I set data in one page (example.com/foo) and retrieve it in another page on the same domain (example.com/bar)?

**Comment by mezza** [comment]
It's a straightforward Key-value store IIRC. There is no page context.

**Comment by cmeeren** [comment]
Does that mean that the keys are accessible even across domains? E.g. that example1.com can set the key `baz` and example2.com can read it?

**Comment by mezza** [comment]
No

**Comment by cmeeren** [comment]
Then does it mean that key `baz` stored on example.com/foo is visible from example.com/bar? You're sure?

**Comment by mezza** [comment]
Yes, you can read key `bad` from example.com/foo and example.com/bar
